dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Helper.cpp

```diff
@@ -86,3 +86,31 @@ HRESULT GetDefaultPriFile(winrt::hstring& filePath)
     bool isPackaged = (hr != HRESULT_FROM_WIN32(APPMODEL_ERROR_NO_PACKAGE));
     return GetDefaultPriFileForCurentModule(isPackaged, filePath);
 }
+
+bool IsWellFormedLanguageTag(const wchar_t* languageTag)
```

Comment:
can use https://learn.microsoft.com/en-us/windows/win32/api/bcp47mrm/nf-bcp47mrm-iswellformedtag

Reply:
done. I don't have this header, so I reused the logic to load the DLL manually as in platform.h/.cpp. Tested. Works as expected.

Reply:
the header should be in SDK. The LoadLibrary code was written before it's added into SDK.

Reply:
according to the docs it should be there since Windows 10 Build 17763, but I don't have it on machine (OS version 10.0.22631 Build 22631), VS 22 installed with multiple SDK versions. I double-checked on another machine with the same software versions

Reply:
I double-checked again, the header is present in SDK v10.0.20348.0 (and later), even though docs says it should be there before

Reply:
I believe bumping WindowsTargetPlatformMinVersion for this specific project is not an option?

Reply:
I pushed the change (including WindowsTargetPlatformVersion bump). I'll revert if it's not ok